tooling/Microsoft.VisualStudio.BlazorExtension/BlazorPackage.cs

```diff
@@ -4,15 +4,37 @@
 using System;
 using System.Runtime.InteropServices;
 using Microsoft.VisualStudio.Shell;
+using Microsoft.VisualStudio.Shell.Interop;
 
 namespace Microsoft.VisualStudio.BlazorExtension
 {
     // We mainly have a package so we can have an "About" dialog entry.
     [PackageRegistration(UseManagedResourcesOnly = true)]
     [AboutDialogInfo(PackageGuidString, "ASP.NET Core Blazor Language Services", "#110", "112")]
     [Guid(BlazorPackage.PackageGuidString)]
+    [ProvideAutoLoad(UIContextGuids80.SolutionExists)]
```

Comment:
I think this is good enough for now. In general this is something to avoid, but the solution to avoiding it is fairly complex, and we'd only bother if we're really productionizing this feature.